src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/hearings/hearingdetails/HearingDetailsService.java

```diff
@@ -25,24 +26,32 @@ public void initialiseHearingDetails(CaseData caseData) {
+        caseData.setHearingDetailsPartHeard(nonNull(selectedListing.getHearingPartHeard()));
+        caseData.setHearingDetailsReservedJudgment(nonNull(selectedListing.getHearingReservedJudgement()));
+        caseData.setHearingDetailsAttendeeClaimant(nonNull(selectedListing.getAttendeeClaimant()));
+        caseData.setHearingDetailsAttendeeNonAttendees(nonNull(selectedListing.getAttendeeNonAttendees()));
+        caseData.setHearingDetailsAttendeeRespNoRep(nonNull(selectedListing.getAttendeeRespNoRep()));
+        caseData.setHearingDetailsAttendeeRespAndRep(nonNull(selectedListing.getAttendeeRespAndRep()));
+        caseData.setHearingDetailsAttendeeRepOnly(nonNull(selectedListing.getAttendeeRepOnly()));
+        caseData.setHearingDetailsTimingStart(nonNull(selectedListing.getHearingTimingStart()));
+        caseData.setHearingDetailsTimingBreak(nonNull(selectedListing.getHearingTimingBreak()));
+        caseData.setHearingDetailsTimingResume(nonNull(selectedListing.getHearingTimingResume()));
+        caseData.setHearingDetailsTimingFinish(nonNull(selectedListing.getHearingTimingFinish()));
+        caseData.setHearingDetailsTimingDuration(nonNull(selectedListing.getHearingTimingDuration()));
+        caseData.setHearingDetailsHearingNotes2(nonNull(selectedListing.getHearingNotes2()));
+    }
 
-        caseData.setHearingDetailsStatus(selectedListing.getHearingStatus());
-        caseData.setHearingDetailsPostponedBy(selectedListing.getPostponedBy());
-        caseData.setHearingDetailsCaseDisposed(selectedListing.getHearingCaseDisposed());
-        caseData.setHearingDetailsPartHeard(selectedListing.getHearingPartHeard());
-        caseData.setHearingDetailsReservedJudgment(selectedListing.getHearingReservedJudgement());
-        caseData.setHearingDetailsAttendeeClaimant(selectedListing.getAttendeeClaimant());
-        caseData.setHearingDetailsAttendeeNonAttendees(selectedListing.getAttendeeNonAttendees());
-        caseData.setHearingDetailsAttendeeRespNoRep(selectedListing.getAttendeeRespNoRep());
-        caseData.setHearingDetailsAttendeeRespAndRep(selectedListing.getAttendeeRespAndRep());
-        caseData.setHearingDetailsAttendeeRepOnly(selectedListing.getAttendeeRepOnly());
-        caseData.setHearingDetailsTimingStart(selectedListing.getHearingTimingStart());
-        caseData.setHearingDetailsTimingBreak(selectedListing.getHearingTimingBreak());
-        caseData.setHearingDetailsTimingResume(selectedListing.getHearingTimingResume());
-        caseData.setHearingDetailsTimingFinish(selectedListing.getHearingTimingFinish());
-        caseData.setHearingDetailsTimingDuration(selectedListing.getHearingTimingDuration());
-        caseData.setHearingDetailsHearingNotes2(selectedListing.getHearingNotes2());
+    private String nonNull(String value) {
+        if (isNullOrEmpty(value)) {
+            return " ";
+        } else {
+            return value;
+        }
```

Comment:
Minor: Would be neater done like this:
`return isNullOrEmpty(value) ? "" : value;`